Do path lookup for 'gs' binary. Sometimes, ./configure doesn't pick up a full
pathname for it.

Needs to be submitted upstream.

=== modified file 'filter/gstoraster.c'
--- filter/gstoraster.c	2015-07-04 11:28:42 +0000
+++ filter/gstoraster.c	2016-10-13 22:14:29 +0000
@@ -420,7 +420,7 @@
     }
 
     /* Execute Ghostscript command line ... */
-    execve(filename, gsargv, envp);
+    execvp(filename, gsargv);
     perror(filename);
     goto out;
   }

